modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/TopLevelBuild.java

```diff
@@ -329,11 +334,23 @@ public Element getValidationGitHubMessageElement() {
 		return validationBuild.getGitHubMessageElement();
 	}
 
+	public boolean hasPullRequestObject() {
```

Comment:
Is this method really necessary? It seems strange that this method exists instead of `getPullRequest()`